eogrow/pipelines/rasterize.py

```diff
@@ -79,44 +43,79 @@ class Schema(Pipeline.Schema):
             description="The storage manager key pointing to the input folder for the rasterization pipeline."
         )
         output_folder_key: str = Field(
-            description="The storage manager key pointing to the output folder for the rasterization pipeline pipeline."
+            description="The storage manager key pointing to the output folder for the rasterization pipeline."
         )
         vector_input: Union[Feature, str] = Field(
             description="A filename in the input_data folder or a feature containing vector data."
         )
         dataset_layer: Optional[str] = Field(
-            description="Name of a layer with data to be rasterized in a multi-layer file."
+            description="In case of multi-layer files, name of the layer with data to be rasterized."
         )
-        columns: List[VectorColumnSchema]
-        preprocess_dataset: Optional[Preprocessing] = Field(
+        raster_value: Optional[float] = Field(
+            description="Value to be used for all rasterized polygons. Use either this or `raster_values_column`."
+        )
+        raster_values_column: Optional[str] = Field(
             description=(
-                "Parameters used by self.preprocess_dataset method. If set to `None` it skips the dataframe preprocess"
-                " step."
+                "GeoPandas dataframe column from which to read rasterization values for each geometry. Use either this"
+                " or `raster_value`."
             )
         )
+        raster_feature: Feature = Field(description="A feature which should contain the newly rasterized data.")
+        polygon_buffer: float = Field(
+            0, description="The size of polygon buffering to be applied before rasterization."
+        )
+        resolution: Optional[float] = Field(
+            description="Rendering resolution in meters. Cannot be used with `raster_shape`."
+        )
+        raster_shape: Optional[Tuple[int, int]] = Field(
+            description="Shape of resulting raster image. Cannot be used with `resolution`."
+        )
+        overlap_value: Optional[int] = Field(description="Value to write over the areas where polygons overlap.")
+        dtype: np.dtype = Field(np.dtype("int32"), description="Numpy dtype of the output feature.")
+        no_data_value: int = Field(0, description="The no_data_value argument to be passed to VectorToRasterTask")
+
+        preprocess_dataset: Optional[Preprocessing] = Field(
+            description="Parameters used by `self.preprocess_dataset` method. Skipped if set to `None`."
+        )
         compress_level: int = Field(1, description="Level of compression used in saving EOPatches")
 
+        _parse_dtype = field_validator("dtype", parse_dtype, pre=True)
+        _check_raster_value_values_column = ensure_exactly_one_defined("raster_value", "raster_values_column")
+        _check_shape_resolution = ensure_exactly_one_defined("raster_shape", "resolution")
```

Comment:
Not sure how common the other option is. I would say the resolution would be sufficient, but I'm not sure if the reason was an actual use case or premature generalization.

In any case, an eopatch should now (or soon) always exist and have a bbox, therefore the resolution parameter should enough.

Reply:
Upon second glance I'd rather leave this as is for now, because tests are affected. The stats are calculated with the shape option, so I would need to update the stats, better to do this in the separate MR

Reply:
:+1: